cloud/docker/_docker.py

```diff
@@ -26,6 +26,7 @@
 module: docker
 version_added: "1.4"
 short_description: manage docker containers
+deprecated: In 2.2 use docker_container and docker_image instead.
```

Comment:
should that be M(docker_container) and M(docker_image)?

I note we haven't done that before though, that would make it a bit easier for someone to jump to documentation for the newer modules